vioscsi/vioscsi.h

```diff
@@ -45,6 +45,10 @@ typedef struct VirtIOBufferDescriptor VIO_SG, *PVIO_SG;
 #define VIRTIO_SCSI_CDB_SIZE   32
 #define VIRTIO_SCSI_SENSE_SIZE 96
 
+#ifndef NTDDI_WINTHRESHOLD
+#define NTDDI_WINTHRESHOLD                  0x0A000000  /* ABRACADABRA_THRESHOLD */
```

Comment:
:)

Reply:
This part is needed to make sure that the code can be compiled successfully  for Vista targets, which is building with old 
WDK tools. WinDDK 7600, which we use in this case, has Windows7 as the most advanced version.